frontend/lib/map/map/map.dart

```diff
@@ -31,23 +31,21 @@ class Map extends StatefulWidget {
 class _MapState extends State<Map> {
   MapboxMapController _controller;
   MapboxMap _mapboxMap;
-  bool _initialLocationUpdateStillPending = true;
 
   @override
   void didChangeDependencies() {
     final config = Configuration.of(context);
     _mapboxMap = new MapboxMap(
       initialCameraPosition: CameraPosition(
-          target: Map.initialLocation, zoom: Map.initialZoomLevel),
+          target: Map.initialLocation, zoom: Map.userLocationZoomLevel),
       styleString: config.mapStyleUrl,
       myLocationEnabled: widget.myLocationEnabled,
       myLocationTrackingMode: widget.myLocationEnabled
           ? MyLocationTrackingMode.Tracking
           : MyLocationTrackingMode.None,
       onMapCreated: _onMapCreated,
       onMapClick: _onMapClick,
-      onUserLocationUpdated: (location) => _onUserLocationUpdated(),
-      onCameraTrackingDismissed: _onCameraTrackingDismissed,
+      compassViewPosition: CompassViewPosition.BottomRight,
```

Comment:
If an acceptance store is selected, the compass is covered by the overlay then, right?

Reply:
Yes